src/lib/actionrunner.js

```diff
@@ -32,20 +32,21 @@ const OPENWHISK_DEFAULTS = {
     // note: openwhisk deployments might have their own versions
     kinds: {
         // "nodejs"     : "openwhisk/nodejsaction:latest", // deprecated, image no longer available
-        "nodejs:6"   : "openwhisk/nodejs6action:latest",
-        "nodejs:8"   : "openwhisk/action-nodejs-v8:latest",
-        "nodejs:10"  : "adobeapiplatform/adobe-action-nodejs-v10:3.0.21", // see: https://hub.docker.com/r/adobeapiplatform/adobe-action-nodejs-v10/tags
-        "python"     : "openwhisk/python2action:latest",
-        "python:2"   : "openwhisk/python2action:latest",
-        "python:3"   : "openwhisk/python3action:latest",
+        // "nodejs:6"   : "openwhisk/nodejs6action:latest",
+        // "nodejs:8"   : "openwhisk/action-nodejs-v8:latest",
+        "nodejs:10"  : "adobeapiplatform/adobe-action-nodejs-v10:3.0.22", // see: https://hub.docker.com/r/adobeapiplatform/adobe-action-nodejs-v10/tags
+        "nodejs:12"  : "adobeapiplatform/adobe-action-nodejs-v12:3.0.22", // see: https://hub.docker.com/r/adobeapiplatform/adobe-action-nodejs-v12/tags
+        // "python"     : "openwhisk/python2action:latest",
+        // "python:2"   : "openwhisk/python2action:latest",
+        // "python:3"   : "openwhisk/python3action:latest",
         // "swift"      : "openwhisk/swiftaction:latest", // deprecated, image no longer available
-        "swift:3"    : "openwhisk/swift3action:latest",   // deprecated, but still available
```

Comment:
why are we removing all of these?

Reply:
See PR description: disable all unsupported kinds in IO Runtime.

These were from raw openwhisk. But Runtime does not support any of those kinds.

Reply:
I see now. I saw your comment in the PR but I was confused since the comment on some of the deleted kinds said `// deprecated, but still available`.